### PR TITLE
Pipelines Update

### PR DESCRIPTION
**Pull request by jduimovich**
This PR contains the following updates. 

- update to using pipelines  bundle v0.1.2 
- pipelines enable build for quay.io for docker and devfile builds
- adds new base cluster tasks for build phases 
- utility tasks for showing pvc contents
- updates to pruning configured (WIP)

### CODE REVIEW

**Comment by ralphbean** [comment]
Relates to https://github.com/redhat-appstudio/build-definitions/pull/8

File: components/build/clustertasks/analyze-devfile.yaml [hunk 1]
```diff
@@ -16,7 +16,7 @@ spec:
     - name: source
   steps:
     - name: analyze-devfile
-      image: quay.io/redhat-appstudio/appstudio-utils:v0.1.1
+      image: quay.io/redhat-appstudio/appstudio-utils:v0.1.2
```
Comment (sbose78) [inline components/build/clustertasks/analyze-devfile.yaml:19]: confirmed that this is available on the registry.

**Comment by sbose78** [comment]
> update to using pipelines bundle v0.1.2

Hi @jduimovich , 

Give that you've updated the consumption of the pipelines in the `hack` directories, could you also update the https://github.com/redhat-appstudio/infra-deployments/tree/main/components/has/.tekton manifests to consume what you have ?

**Comment by sbose78** [comment]
Can be done in a separate PR though.

**Pull request merged**